File: src/main/Java/boj_step13/boj2750.java
import java.io.*;
import java.util.ArrayList;
import java.util.Collections;

public class boj2750 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        ArrayList<Integer> list = new ArrayList<>();
        while (n-- > 0) list.add(Integer.parseInt(reader.readLine()));
        Collections.sort(list);
        for (int i: list) writer.append(i + "\n");
        writer.close();
    }
}

File: src/main/boj_step13/boj2750.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine().toInt()
    var arr = mutableListOf<Int>()
    repeat(n) {
        arr.add(r.readLine().toInt())
    }
    arr = arr.sorted().toMutableList()
    arr.forEach { w.write("$it\n") }
    w.close()
}